source/extensions/filters/http/jwt_authn/jwks_cache.cc

```diff
@@ -92,6 +102,8 @@ class JwksDataImpl : public JwksCache::JwksData, public Logger::Loggable<Logger:
   TimeSource& time_source_;
   // The pubkey expiration time.
   MonotonicTime expiration_time_;
+  // The JwtCache object
+  std::unique_ptr<JwtCache> jwt_cache_;
```

Comment:
just embed JwtCache here as
```
    JwtCache  jwt_cache_
``

In the JwksDataImpl constructor add
```
     :jwt_cache_(provider.jwt_cache_enabled(),  provider.jwt_cache_size())
```

Reply:
Due to some gcc error need to use unique ptr

Reply:
PTAL